roles/agnosticv/templates/resource_provider.yml.j2

```diff
@@ -65,6 +65,13 @@ spec:
     - add
     - replace
     pathMatch: /spec/vars/desired_state
+{% if merged_vars.__meta__.catalog.parameters | default([]) | length > 0 %}
+{%   for parameter in merged_vars.__meta__.catalog.parameters %}
+{%     if allowUpdate | default(false) | bool %}
```

Comment:
This would be `parameter.allowUpdate` I believe.

Reply:
whoops!

fixed